lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -616,6 +617,13 @@ private function determineIdGeneratorStrategy(AbstractPlatform $platform): int
             }
         }
 
+        $nonIdentityDefaultStrategy = self::NON_IDENTITY_DEFAULT_STRATEGY;
```

Comment:
What's happening with `$nonIdentityDefaultStrategy` and where is it used? What am I missing?

Reply:
Sorry, I forgot to use it below :facepalm:

Reply:
I added tests.